src/pages/signin/ResendValidationForm.js

```diff
@@ -40,14 +64,15 @@ class ResendValidationForm extends React.Component {
                     </Text>
                 </View>
                 <View style={[styles.mt4]}>
-                    <SubmitButton
+                    <ButtonWithLoader
                         text="Resend Link"
-                        isLoading={this.state.isLoading}
+                        isLoading={this.props.account.loading}
                         onClick={this.validateAndSubmitForm}
                     />
+                    <ChangeExpensifyLoginLink />
                 </View>
 
-                {this.state.formSuccess && (
+                {!_.isEmpty(this.state.formSuccess) && (
```

Comment:
Just curious, but why did this need to be changed from the string check? 

I _thought_ I saw something about how `empty string && render something` will try to render the empty string on native platforms which is bad.

Reply:
Yes that must be it. I wonder if there is a way to get the linter to enforce this... 🤔

Reply:
Maybe just switch to ternaries for everything and ban short circuit 🤷

Reply:
Yeah, this is specific to using string values and `&&` which breaks mobile. It would be nice to enforce this with a linter somehow, but it's not possible today. I'd hate to lose the ability to use the `&&` shortcut because ternarys make the code much less clean IMO (eg. seeing `: null` everywhere is tiresome and noisy).